src/main/scala/org/zalando/spearheads/innkeeper/services/RoutesService.scala

```diff
@@ -19,7 +19,7 @@ trait RoutesService {
              createdBy: String,
              createdAt: LocalDateTime = LocalDateTime.now()): Future[Result[RouteOut]]
 
-  def remove(id: Long): Future[Result[RouteOut]]
+  def remove(id: Long): Future[Result[Boolean]]
```

Comment:
Actually Boolean is random type put there... I always put true. In case there is a failure, we have the Failure case.